exercicios/para-casa/entregas/laura-leamari/projeto/CurrentAccount.js

```diff
@@ -0,0 +1,32 @@
+const { BankAccount } = require('./BankAccount');
+
+class CurrentAccount extends BankAccount{
+	#balance = 0;
+
+  transferTo(anotherAccount, amount){
+    if (!(anotherAccount instanceof BankAccount)) {
+			console.log('Informe uma conta válida!');
+			return;
+		}
+
+    let amountToBeDebited = amount;
+
+    if (this.#balance >= amountToBeDebited) {
+			this.#balance -= amountToBeDebited;
```

Comment:
Primeiramente, seu codigo está bem estruturado e funcional. Pouca coisa tenho o que te dar como dica de ajustes: Nessa linha em questao, eu so ajustaria o atributo privado. Não precisa colocar ele, pq ele já é herdado da classe pai e vc pode acessá-lo diretamente. Ficando assim:  
 if (this.balance >= amountToBeDebited) {
      this.balance -= amountToBeDebited;
      anotherAccount.balance += amount;